isa_helper.c

```diff
@@ -4,9 +4,7 @@
 uint32_t rotate_right(uint32_t shiftee, uint8_t shifter)
 {
     return (shiftee << (32 - shifter)) | (shiftee >> (shifter));
-}
-
-uint8_t get_bit(uint32_t from, uint8_t bitid)
+} uint8_t get_bit(uint32_t from, uint8_t bitid)
```

Comment:
yah but it is in your commit so fuck off
you must have deleted this lines sometime and didn't even care to check git diff before pushing